### ISSUE TITLE
v1.9.2.9 pays 20% less compared to v1.9.1.9

### ISSUE THREAD

**Issue by gdcfxvnjvbncvb** [nicehash/NiceHashMiner#1690]
Hello,

Why does the new platform pay 20% less compared to the old platform ?

Here is an image, at the top the old platform (v1.9.1.9), at the bottom the new platform (v1.9.2.9).
Same rig and graphic card.

![003](https://user-images.githubusercontent.com/24574467/61621872-ff8a9700-ac73-11e9-9752-a7e8f69a75f7.png)

**Comment by Twigho** [comment]
Hello,
0.13239-20%=0.105912
0.13239-4%=0.1270944 (2%third party dev fee + 2% NiceHash)
0.13239-5%=0.1257705 (if 3% thrid party dev fee)

NBMiner dev Fee (In the paper) : 
 * tensority_ethash 3%
  * tensority(Pascal) 2%, tensority(Turing) 3%
  * ethash 0.65%
  * cuckaroo & cuckarood & cuckatoo & cuckoo_ae & cuckaroo_swap 2%
  * progpow_sero 2%

Maybe NBMiner fees are not respected... Or it is the estimation of the btc value. You can see the 1btc=xxusd if you put your cursor on the 3 black bubuls on the right bottom. (this value can be wrong)

**Comment by gdcfxvnjvbncvb** [comment]
I came out with the 20% difference, comparing the "EUR/Day" value.
You are correct, the estimation of the BTC value isn't the same between v1.9.1.9 (9125 EUR/BTC) and v1.9.2.9 (8316 EUR/BTC).

You are correct a second time, it is a 5% difference, if we compare the mBTC/Day" value.
That would mean that the fees are taken into account and deducted in the "mBTC/Day" value in v1.9.2.9, but not in v1.9.1.9 ?

**Comment by Twigho** [comment]
Indeed, but that is my theory. this gives a more realistic estimate...
Maybe you should test with another miner (like gminer). If you have 4% it means that nbminer take 3% instead of 2% as written in their "readme"..

**Comment by Maas1337** [comment]
same here, switched back to 1.9.1.9

**Comment by jwesolo** [comment]
The old platform (1.9.1.x) is more profitable than the new platform (1.9.2.x) for some algorithms because they are two different marketplaces where buyers are paying different amounts for hashing power. This is not a software issue.

**Comment by Johann1982** [comment]
I am getting very bad hashrates with the latest nicehash version. I moved over to the new platform yesterday and installed the new software. Now on GrinCukatoo31 NBminer I am only getting 0.568G/s where I used to get 1.4G/s. GTX 1080Ti Founders edition. When I run NiceHash Miner Legacy v1.9.0.20 with GrinCuckatoo31 I get 1.440 G/s. I even tried copying the NBminer from 3rd party folder to the new nicehash NBminer folder. For some reason nhm 1.9.2.10 only gets around 0.5 G/s. Something in the new nicehash does not work properly.
